schemas/erc20/schema.json

```diff
@@ -0,0 +1,49 @@
+{
+  "schema": "ERC-20",
+  "description": "Table definitions for the ERC-20 schema. These tables can be used for any protocol.",
+  "protocolCategory": ["Bond", "Gaming", "LRT", "Oracle", "Payment", "Quest", "SocialFi", "Stablecoin", "Wallet", "Other"],
+  "version": "1.0.0-alpha",
+  "tables": [
+    {
+      "label": "Token ERC-20 Transfers",
+      "tableName": "general_token_erc20_transfers",
+      "aggregation": "transaction",
+      "uniqueKey": ["user_address", "token_address", "block_date"],
+      "description": "User level token balance snapshots.",
+      "properties": {
```

Comment:
Let's add `block_date`, `transaction_hash`, and `log_index`

Need the last 2 for a uniqueKey, and the first for partitions

Reply:
`timestamp` = `block_date` here.  Does that work okay or do I need to change the name?

Reply:
Actually, not needed, this setup works